### ISSUE TITLE
Backup Library Issue

### ISSUE THREAD

**Issue by NephyMephy** [Aidoku/Aidoku#287]
Similar to issue #110 
When importing a backup to a new Aidoku installation such as sideload to testflight migration, Aidoku fails to load chapters on bookmarked content even after installing sources and source lists. 
The issue can be be fixed after unbookmarking and bookmarking again. However it is very problematic due to the absurd number of manhwas I have bookmarked haha.
![IMG_5001 (2)](https://github.com/Aidoku/Aidoku/assets/58957768/ad7a889b-ea27-472c-9947-506bfa6718ca)

Steps to Reproduce:
1. Import back up to new installation of Aidoku
2. Reinstall Source list and Sources
3. Try to access bookmarked content 
4. Fails to load chapters and progress

Steps for Workaround: (Video Linked)
1. Follow steps 1 and 2 from above
2. Unbookmark your bookmarked content
3. Bookmark it again
4. Repeat until all of your library is updated

https://github.com/Aidoku/Aidoku/assets/58957768/0cee4efb-1f1c-4184-a21c-2c3e954ab89b

**Comment by Skittyblock** [comment]
does this only happen with asura scans? I think it might just be because they change their ids so often so you need to migrate

**Comment by NephyMephy** [comment]
I believe the same issue appeared in MangaSee. It happened with all  Scan groups in my library as seen in the video. Oshi no Ko is not scanned by Asura Scans for example.

**Comment by Skittyblock** [comment]
alright I'll look into it, if you could send me your backup that'd be helpful

**Comment by NephyMephy** [comment]
> alright I'll look into it, if you could send me your backup that'd be helpful
[aidoku_2023-07-26_00-24-57_1.aib.zip](https://github.com/Aidoku/Aidoku/files/12207558/aidoku_2023-07-26_00-24-57_1.aib.zip)
Sorry, Github doesn't seem to support uploading aib files.

**Comment by Skittyblock** [comment]
I installed your backup and it worked perfectly fine.

Then I went back and read your steps and realized your issue: you didn't have the sources installed when you restored the backup. There isn't any way for aidoku to load the chapters from the source if you don't actually have it installed when restoring. You either need to install the sources and restore the backup again, or just pull to refresh your library.
